Benchmarks/Thresholds/5.10/SwiftKafkaConsumerBenchmarks.SwiftKafkaConsumer_basic_consumer_messages_1000.p90.json

```diff
@@ -0,0 +1,10 @@
+{
+  "allocatedResidentMemory" : 77266944,
+  "cpuTotal" : 200000000,
+  "objectAllocCount" : 5549,
+  "releaseCount" : 15168,
+  "retainCount" : 7108,
+  "retainReleaseDelta" : 2511,
+  "throughput" : 2,
+  "wallClock" : 695307500
+}
```

Comment:
In most of our repositories we only add benchmarks for `mallocCountTotal` because it is a stable measure. We don't yet have dedicated runners which would be required for time-based benchmarks

see e.g. https://github.com/apple/swift-nio/blob/main/Benchmarks/Thresholds/5.9/NIOCoreBenchmarks.NIOAsyncChannel.init.p90.json